src/lib/docs/Antelope/Transaction.md

```diff
@@ -141,13 +140,13 @@ const transaction = Transaction.from({
 })
 ```
 
-The result stored as `transaction` will be an instance of `Transaction` and be ready to sign.
+The result stored as a `transaction` will be an instance of `Transaction` and be ready to sign.
```

Comment:
I think the original might be correct here, since `transaction` is the name of the Transaction being stored.

Reply:
fixed